include/oneapi/dpl/pstl/parallel_backend_tbb.h

```diff
@@ -1308,35 +1308,46 @@ __parallel_for_each(oneapi::dpl::__internal::__tbb_backend_tag, _ExecutionPolicy
 
 namespace __detail
 {
-struct __get_num_threads
+
+// Workaround for VS 2017: declare an alias to the CRTP base template
+template <typename _ValueType, typename... _Args>
+struct __enumerable_thread_local_storage;
+
+template <typename... _Ts>
+using __etls_base = __utils::__enumerable_thread_local_storage_base<__enumerable_thread_local_storage, _Ts...>;
+
+template <typename _ValueType, typename... _Args>
+struct __enumerable_thread_local_storage : public __etls_base<_ValueType, _Args...>
 {
-    std::size_t
-    operator()() const
+
+    template <typename... _LocalArgs>
+    __enumerable_thread_local_storage(_LocalArgs&&... __args)
+        : __etls_base<_ValueType, _Args...>({std::forward<_LocalArgs>(__args)...})
```

Comment:
I don't understand the benefit of wrapping the `__args` pack in braces on the way in to `__etls_base`. 
Can you explain why this is a benefit?

Reply:
Nevermind, I see how the base has changed to accept a tuple of `__args...` rather than a pack of `__args` and then constructing a tuple.  I missed that change.  Now it makes sense.